### DISCUSSION TITLE
能不能传一些视频

### DISCUSSION THREAD

**Discussion by chizi-chizi** [WeNeedHome/SummaryOfLoanSuspension#513] [Ideas]
看到抖音上有业主和银行对质的视频， 意思是地方房管局的问题， 链接是：5.66 WZM:/ 复制打开抖音，看看【赵哥侃世界的作品】让子弹飞一会儿 https://v.douyin.com/2jHrMnH/

**Comment by kydyahah** [comment]
最好把710荷兰阿姆斯特丹银行门口的视频也传上来

**Reply by zcaaron** [reply]
应该传上来

**Reply by chizi-chizi** [reply]
我不会弄。。。
